scripts/calico.yaml

```diff
@@ -145,19 +150,27 @@ spec:
     spec:
       hostNetwork: true
       tolerations:
+      # this taint is set by all kubelets running `--cloud-provider=external`
```

Comment:
We don't really need these because we always use --cloud-provider=aws, but there is no problem with keeping it if it is just a copy/paste from upstream.